paddle/fluid/primitive/primitive.yaml

```diff
@@ -3,6 +3,7 @@
 - multiply
 - divide
 - elementwise_pow
+- rsqrt
```

Comment:
Does vjp rule of `rsqrt` need to be defined?

Reply:
yes, to do later